lib/tasks/export.rake

```diff
@@ -0,0 +1,11 @@
+# Tasks to export per-entity CSV files for CCS BI
+namespace :export do
+  desc 'Export task entities to CSV'
+  task tasks: :environment do
+    filename = "/tmp/tasks_#{Time.zone.today}.csv"
+    warn("Exporting tasks to #{filename}")
```

Comment:
what's the thinking behind output this to STDERR?

Reply:
I have a habit of telling people what I'm doing in anything console-based. This was `$stderr.puts` but Rubocop doesn't like that despite them really not being the [same thing](https://github.com/rubocop-hq/rubocop/issues/4888). Since it doesn't affect STDOUT if we elect to send output there, it's one way of letting operators know (without going to the code) that at present these things are going directly into `/tmp` because we're likely to think about the mechanism of generation via streaming later as part of [transfer automation](https://trello.com/c/DZEc7RwZ/386-automate-the-creation-and-transfer-of-the-export-results-to-a-place-accessible-by-ccs).

It has a nasty side-effect of spitting out a few lines during testing. I'm not keen on this. Mechanisms I've used in the past to avoid it have included making our own [wrapper for puts](https://github.com/alphagov/transition/blob/master/lib/transition/import/console_job_wrapper.rb), which seems a bit wheel-reinvent-y in several years' hindsight. If you know of a good way with standard rails logging please let me know! Always keen to be a good shell citizen.

Reply:
👍🏽